ftl/node/benchmark/small_app.yaml

```diff
@@ -0,0 +1,17 @@
+# This is a 'small' node app, with 1 dependency (20th percentile of all node projects with dependencies)
+steps:
+# Build benchmark image with bazel
+  - name:  'gcr.io/cloud-builders/bazel'
+    args: ['run','ftl/node/benchmark:node_benchmark_image', '--', '--norun']
+# Clone git repo
+  - name:  'gcr.io/cloud-builders/git'
+    args: ['clone','https://github.com/KevinGaudin/monsterid.js.git']
```

Comment:
We should probably host the sample apps so that there aren't unexpected changes in the app, for licensing purposes, and also in case the repo is deleted.

Reply:
oh good point, where should we host them?

Reply:
Yeah it would be easier if we can find ones we already host with the same number of deps, then commit just the package.jsons to a directory in this repo